### ISSUE TITLE
increase batch size to remove for loop which is inefficient

### ISSUE THREAD

**Issue by hvgazula** [neuronets/nobrainer_training_scripts#3]
https://github.com/neuronets/nobrainer_training_scripts/blob/c985279985771e424ac60347dedd6d158204b099/1.2.0/callbacks.py#L30
and
https://github.com/neuronets/nobrainer_training_scripts/blob/c985279985771e424ac60347dedd6d158204b099/1.2.0/callbacks.py#L45

**Comment by hvgazula** [comment]
While I wrote my custom prediction script inside a callback, ideally we should be using this https://github.com/neuronets/nobrainer/blob/976691d685824fd4bba836498abea4184cffd798/nobrainer/prediction.py#L15 to be able to extend to all models.

**Comment by satra** [comment]
isn't predict already in the api? for prediction inside a callback, isn't that also happening in the validation callback?

**Comment by hvgazula** [comment]
yes, the `predict` is available (as pointed out in the previous comment). I put aside a test set and was quickly prototyping things and am now integrating it into the API, hence some comments. At the time of prototyping, the predict method (as I saw it) was wrapped inside the nobrainer model object and I didn't want to wait until I understood all of that. Also, I wanted to control some outputs as well. For example, saving both ground truth and predictions as pngs ala the images I shared with you on slack.
